### PR TITLE
Homework5: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Hello, Sergei,
My second PR is ready for review

### CODE REVIEW

**Comment by Stml89** [comment]
Strange, but PR title job passed but no comment from it =(

File: Homework5.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+# Пример использования
+input_word = "Hello"
+output_word = add_ing(input_word)
+print(output_word)
+
+# В строке “Ivanou Ivan” поменяйте местами слова => "Ivan Ivanou"
+name = "Ivanou Ivan"
+new_name = name[7:11] + " " + name[0:6]
```
Comment (Stml89) [inline Homework5.py]: use `name.split()` instead

**Review by Stml89 — CHANGES_REQUESTED** [review]
Need to fix a comment

**Comment by Pavel2812mi** [comment]
Done, 
Is it ok that auto assign check didn't work now?

**Pull request closed without merging**